src/components/LoanCards/AdaptiveMicroLoanCard.vue

```diff
@@ -0,0 +1,268 @@
+<template>
```

Comment:
This is the "new" minimal loan card. Looks the same on large displays but becomes a row on small devices. Has new events for current use case.